tests/operators/assignmentoperator_test.cpp

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
+#include <cassert>
+
+#include "../../big-int/src/bigint.hpp"
+using namespace libbig;
+
+int main() {
+
+	largeInt bigIntVarA, bigIntVarB, bigIntVarC, bigIntVarD;
+    int intVarA = 1000000;
+    int64_t int64_tVarA = 1000000000000000011;
+
+    //Assigning int value to largeInt variable
+    bigIntVarA = 1928374;
+    assert(bigIntVarA == std::string("1928374"));
+
+    //Assigning one largeInt variable to another largeInt variable
+    bigIntVarB = bigIntVarA;
+    assert(bigIntVarB == std::string("1928374"));
+
+
+    //Assigning int variable to largeInt variable
+    bigIntVarC = intVarA;
```

Comment:
Codacy found an issue: [Variable 'bigIntVarC' is assigned a value that is never used.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6287175)